blog.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>ADOPTADOG | Seattle's premiere dog adoption agency</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <!-- CONTAINER START -->
+  <div class="container">
+
+    <!-- HEADER START -->
+    <header>
+      <div class="logo-header">
+        <img src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo">
+        <p>Seattle's premiere dog adoption agency</p>
+      </div>
+      <div class='checkout-wrapper'>
+        <div>
+          <img src="images/cart-icon.svg" alt="cart icon">
+          <small>$270.50</small>
+        </div>
+        <div>
+          <a href="checkout.html" class="btn btn-checkout">Checkout</a>
+        </div>
+      </div>
+    </header>
+    <!-- HEADER END -->
+
+    <!-- NAV START -->
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">Dogs</a></li>
+        <li><a href="blog.html" class="active">Blog</a></li>
+      </ul>
+    </nav>
+    <!-- NAV END -->
+
+    <!-- SECTION START -->
+    <section class="section-blog">
+
+      <h2>Adoptadog Blog</h2>
+
+      <div class="blog-box">
```

Comment:
You should no longer have these elements in html and only use the script to generate blog posts.

Reply:
I'm not sure if I got to that point yet. Assignment requirements didn't ask me to use scripts to generate blogs. I believe it will be covered in jQuery. Thank you.